pkg/sink/interface.go

```diff
@@ -16,13 +13,6 @@ type Datum interface {
 	ID() string
 }
 
-// Client contains methods to call a gRPC client.
-type Client interface {
```

Comment:
do we have the Client interface for function? I don't see in the change files...

Reply:
Updated